### ISSUE TITLE
OKD support for OpenStack provider

### ISSUE THREAD

**Issue by DanyC97** [openshift/okd#16]
We would like to keep this master issue to link new issues which are related to OpenStack deployment so we can keep track as well as letting community know if a specific provider deployment is supported or has known issues etc

**Comment by vrutkovs** [comment]
Openstack images in `origin` should use RDO instead of OSP so that these could be redistributed

**Comment by logdnalf** [comment]
Any progress on this?

**Comment by vrutkovs** [comment]
It appears we're blocked on https://github.com/coreos/ignition/pull/892 - it needs to land in FCOS so that we could proceed with the install

**Comment by SvenMW** [comment]
is general openstack support planned?
the docs only mention the redhat openstack distribution, not plain openstack.

**Comment by vrutkovs** [comment]
>the docs only mention the redhat openstack distribution, not plain openstack.

Any openstack distro should work, lets file a new ticket to reflect that in the docs

**Comment by bshephar** [comment]
> is general openstack support planned?
> the docs only mention the redhat openstack distribution, not plain openstack.

The only proviso here is that versions prior to 4.4 used Swift, after 4.4 it's using Glance. So you would just need to make sure that your OpenStack environment meets the requirements from the Docs. Such as Swift is available ( if < 4.4 ), Neutron trunk ports, DNS, etc. Other than that, Red Hat OpenStack doesn't have anything special that allows it to work. You should be fine with any deployment of OpenStack.

Have you run into any specific issues with it, or are you just asking in general?

**Comment by vrutkovs** [comment]
Available in 4.5, lets open separate issues if problems occur
